tests/unit/query_graph/sql/test_forward_aggregate.py

```diff
@@ -0,0 +1,28 @@
+"""
+Test target forward aggregate module
+"""
+from featurebyte import SourceType
+from featurebyte.query_graph.sql.common import REQUEST_TABLE_NAME
+from featurebyte.query_graph.sql.forward_aggregate import get_forward_aggregate_sql
+from tests.util.helper import assert_equal_with_expected_fixture
+
+
+def test_get_forward_aggregate_sql(query_graph_with_groupby, update_fixtures):
```

Comment:
`query_graph_with_groupby` contains normal window aggregates and that is probably why the generated sql is still showing tile related queries. I think we should add a new fixture in `tests/unit/query_graph/conftest.py` with a forward aggregate node.